ui/jobs/combo_tracker.ts

```diff
@@ -1,7 +1,19 @@
 import { kAbility, kComboBreakers } from './constants';
 
+type StartMap = Record<string, {
+  id?: string;
+  next?: StartMap;
+}>;
+
 export default class ComboTracker {
-  constructor(comboBreakers, callback) {
+  comboTimer?: number | undefined;
```

Comment:
I think `comboTimer: number | undefined` or `comboTimer?: number`, but no need for both.